package.json

```diff
@@ -119,7 +119,8 @@
       "dist/msa.js"
     ],
     "css": [
-      "css/msa.css"
+      "css/msa.css",
+      "css/font-awesome/css/font-awesome.min.css"
```

Comment:
Please let us try to keep the msa slick and not make that the default without consideration. There is also a project that allows to select a specifically a couple of icons from font awesome